fp.c

```diff
@@ -0,0 +1,30 @@
+#include <stdio.h>
```

Comment:
Why do we need `stdio.h`, as you are not using any `printf`.

Reply:
And also variable names of arguments and inside functions should have meaning full name.